rel/leo_gateway/reltool.config.in

```diff
@@ -95,6 +95,7 @@
        {app, leo_watchdog,          [{incl_cond, include}]},
        {app, savanna_agent,         [{incl_cond, include}]},
        {app, savanna_commons,       [{incl_cond, include}]}
+       %% SD_NOTIFY_PLACEHOLDER
```

Comment:
Please add the CAUTION sentence like
```erlang
%% CAUTION: Don't remove the above line as it will be replaced through make release to generate reltool.config
```